app/helpers/events_helper.rb

```diff
@@ -0,0 +1,16 @@
+module EventsHelper
+  def event_status_options(event)
+    statuses = Event::STATUSES.values
+    if event.public_session_formats.empty? || event.guidelines.blank?
```

Comment:
Please look into combining this logic with the existing logic of event.incomplete_checklist_items as it does conceptually the same thing.